src/modules/previewpane/MonacoPreviewHandler/MonacoPreviewHandlerControl.cs

```diff
@@ -236,20 +198,26 @@ private async void CoreWebView2_NewWindowRequested(object sender, CoreWebView2Ne
         /// </summary>
         public void FormResize(object sender, EventArgs e)
         {
-            _webView.Height = this.Height;
-            _webView.Width = this.Width;
-            this.Update();
+            _webView.Height = Height;
+            _webView.Width = Width;
+            Update();
         }
 
         /// <summary>
         /// This event initializes the webview and sets various settings
         /// </summary>
         [STAThread]
-        private void WebView2Init(object sender, CoreWebView2NavigationCompletedEventArgs e)
+        private async void WebView2Init(object sender, CoreWebView2NavigationCompletedEventArgs e)
         {
             // Checks if already navigated
             if (!_hasNavigated)
             {
+                await _gatherFileInformationTask.ConfigureAwait(true);
+                await _webView.CoreWebView2.ExecuteScriptAsync("editor.setValue(\"" + _fileContent.Replace("\\", "\\\\").Replace("\"", "\\\"").Replace("\n", "\\n").Replace("\r", "\\r") + "\");").ConfigureAwait(true);
+                await _webView.CoreWebView2.ExecuteScriptAsync("monaco.editor.setModelLanguage(editor.getModel(), \"" + _vsCodeLangSet + "\");").ConfigureAwait(true);
+                await _webView.CoreWebView2.ExecuteScriptAsync("editor.updateOptions({\"wordWrap\": \"" + (Settings.Wrap ? "on" : "off") + "\"});").ConfigureAwait(true);
+                await _webView.CoreWebView2.ExecuteScriptAsync("editor.updateOptions({\"theme\": \"" + (Settings.GetTheme() == "dark" ? "vs-dark" : "vs") + "\"});").ConfigureAwait(true);
```

Comment:
Don't wevhave this code on multiple places? I personally don't like duplicated code that can changes multiple times in the future as contributors need find every places.